db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Answer.delete_all
+ApplicationRecord.delete_all
+Keyword.delete_all
+Officer.delete_all
+Question.delete_all
+Rso.delete_all
+
+rso.create!(name: 'Association for Computer Machinery',
+            shortName: 'ACM',
+            category: 'Computer Science')
+
+rso.create!(name: 'Knitting Club',
```

Comment:
I'm slightly confused by the point of the category field under the Rso class. Should this be associated with any keywords? It kind of seems weird to associate a category like "special interest" or "educational" with a keyword, since they're so vague by definition.